scripts/is-release-needed.sh

```diff
@@ -6,6 +6,8 @@ PACKAGE_NAME="$(node -p -e "require('./package.json').name")"
 
 FOUND_VERSION=$(npm view $PACKAGE_NAME versions | grep $PACKAGE_VERSION)
 
+echo "Checking if $PACKAGE_NAME@$PACKAGE_VERSION is already published..."
+
 if [ -z "$FOUND_VERSION" ]; then
     IS_NEW_VERSION=true
 fi
```

Comment:
Instead of returning a string, I think this script should simply `exit 0` on true and `exit 1` on false (or maybe vice versa, whatever makes the logic work properly).